### ISSUE TITLE
unable to handle non-jar artifact types?

### ISSUE THREAD

**Issue by xwb1989** [lgrignon/jsweet-maven-plugin#18]
Hi,

I'm trying to use jsweet in our project and found it's unable to handle 'pom' type dependency. For example, 

        <dependency>
            <groupId>org.eclipse.jetty.aggregate</groupId>
            <artifactId>jetty-all</artifactId>
            <version>9.3.8.v20160314</version>
            <type>pom</type>
        </dependency>

the artifact shall be `jetty-all-9.3.8.v20160314.pom`, but the resolver is looking for: `jetty-all-9.3.8.v20160314.jar` and therefore throwing the error:

```
[ERROR] failed to create transpiler
org.apache.maven.artifact.resolver.MultipleArtifactsNotFoundException: Missing:
----------
1) org.eclipse.jetty.aggregate:jetty-all:jar:9.3.8.v20160314
```

**Comment by lgrignon** [comment]
Yes, there might be some issue in this case because the plugin performs a manual resolve, and some cases may not be covered. 
It should be fairly simple to fix by simply exclude non-jar dependencies. I will take a look at it at the end of the week, if you find time to fix it before, please PR and I will merge.

**Comment by lgrignon** [comment]
Took my time to push but it should fix by now, could you please try the last 1.2.0-SNAPSHOT version of the plugin to see if it is resolved now. If so, could you please close issue :)

**Comment by xwb1989** [comment]
Thank you, it works now!
